composer/trainer/trainer.py

```diff
@@ -261,13 +245,17 @@ def __init__(
             log_destinations = [TQDMLoggerBackend()]
         self.logger = Logger(self.state, log_destinations)
         self.state.callbacks = [*log_destinations, *callbacks]
-
         self.engine = Engine(self.state, self.state.algorithms, self.logger, self.state.callbacks)
 
         self.validate_every_n_batches = validate_every_n_batches
         self.validate_every_n_epochs = validate_every_n_epochs
         self.compute_training_metrics = compute_training_metrics
-        self.grad_clip_norm = grad_clip_norm
+        self._grad_clip_norm = grad_clip_norm
```

Comment:
since this is not a property of state, we can not make this a getter/private attribute